### PR TITLE
Citation file

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  orcid: ""
+- family-names: "Hertweck"
+  given-names: "Kate"
+  orcid: ""
+- family-names: "Irving"
+  given-names: "Damien"
+  orcid: ""
+- family-names: "Milligan"
+  given-names: "Ian"
+  orcid: ""
```
Comment (ianmilligan1) [inline CITATION.cff]: <https://orcid.org/0000-0002-1470-7723>

**Review by ianmilligan1 — APPROVED** [review]
Looks great, thanks Greg - hope you're doing well too!

**Comment by raynamharris** [comment]
Mine is 0000-0002-7943-5650

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- family-names: "Harris"
+  given-names: "Rayna M."
+  orcid: "https://orcid.org/0000-0002-7943-5650"
+- family-names: "Hertweck"
+  given-names: "Kate"
+  orcid: ""
+- family-names: "Irving"
+  given-names: "Damien"
+  orcid: ""
```
Comment (DamienIrving) [inline CITATION.cff]:
```suggestion
  orcid: "https://orcid.org/0000-0003-1258-5002"
```

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    orcid: ""
+  - family-names: "Harris"
+    given-names: "Rayna M."
+    orcid: "https://orcid.org/0000-0002-7943-5650"
+  - family-names: "Hertweck"
+    given-names: "Kate"
+    orcid: ""
+  - family-names: "Irving"
+    given-names: "Damien"
+    orcid: ""
```
Comment (DamienIrving) [inline CITATION.cff]:
```suggestion
    orcid: "https://orcid.org/0000-0003-1258-5002"
```

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  orcid: ""
+- family-names: "Emonet"
+  given-names: "Rémi"
+  orcid: ""
+- family-names: "Harris"
+  given-names: "Rayna M."
+  orcid: "https://orcid.org/0000-0002-7943-5650"
+- family-names: "Hertweck"
+  given-names: "Kate"
+  orcid: ""
```
Comment (k8hertweck) [inline CITATION.cff]:
```suggestion
  orcid: "https://orcid.org/0000-0002-4026-4612"
```

File: CITATION.cff [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    orcid: ""
+  - family-names: "Emonet"
+    given-names: "Rémi"
+    orcid: ""
+  - family-names: "Harris"
+    given-names: "Rayna M."
+    orcid: "https://orcid.org/0000-0002-7943-5650"
+  - family-names: "Hertweck"
+    given-names: "Kate"
+    orcid: ""
```
Comment (k8hertweck) [inline CITATION.cff]:
```suggestion
    orcid: "https://orcid.org/0000-0002-4026-4612"
```

File: CITATION.cff [hunk 6]
```diff
@@ -0,0 +1,56 @@
+cff-version: 1.2.0
+message: "Please cite this paper as shown."
+authors:
+- family-names: "Devenyi"
+  given-names: "Gabriel A."
+  orcid: ""
```
Comment (gdevenyi) [inline CITATION.cff:6]: 0000-0002-7766-1187

Thanks for updating this!

**Pull request merged**